src/platform/backends/lxd/lxd_virtual_machine_factory.cpp

```diff
@@ -67,7 +68,28 @@ void mp::LXDVirtualMachineFactory::prepare_instance_image(const mp::VMImage& /*
 
 void mp::LXDVirtualMachineFactory::hypervisor_health_check()
 {
-    auto reply = lxd_request(manager.get(), "GET", base_url);
+    QJsonObject reply;
+
+    try
+    {
+        reply = lxd_request(manager.get(), "GET", base_url);
+    }
+    catch (const LocalSocketConnectionException& e)
+    {
+        auto error_code = e.get_error();
+
+        if (error_code == QLocalSocket::ConnectionRefusedError)
+        {
+            throw std::runtime_error(
+                fmt::format("{}\nPlease ensure the LXD snap interface is connected via `snap connect "
+                            "multipass:lxd`\nand that the LXD snap is enabled and running.",
```

Comment:
This needs to be `… lxd`, otherwise:
```
error: snap "core" has no "lxd" interface slots
```

Reply:
Right, will fix.

Reply:
Ok, fixed.  Let me know if you want it tweaked (or propose what you want :smiley: )